versions.tf

```diff
@@ -1,3 +1,10 @@
 terraform {
   required_version = ">= 0.12"
```

Comment:
can this be changed to at least >=0.13?

Reply:
It can be, but I don't think it's necessary because the module is not using features specific to 0.13.